### PR TITLE
add image scanning section

### PR DESCRIPTION
**Pull request by aweiteka**
Preview link: https://people.redhat.com/aweiteka/docs/preview/20170512/index.html#scanning

### CODE REVIEW

**Comment by adahms** [comment]
Thanks for the PR, @aweiteka!

Assigning @dayleparker as a reviewer.

File: doc-QuickStart_OpenShift_Provider_Guide/topics/image-scanning.adoc [hunk 1]
```diff
@@ -0,0 +1,79 @@
+== Configuring Image Scanning
+
+{product-title} manages vulnerability scanning of container images. When the OpenShift provider is added, OpenShift images from the internal registry are discovered. To enable image scanning, perform the following configuration steps:
+
+. Navigate to menu:Compute[Containers > Providers].
+. Select the *OpenShift* provider checkbox.
```
Comment (dayleparker) [inline doc-QuickStart_OpenShift_Provider_Guide/topics/image-scanning.adoc]: To be a little more clear, I'd suggest writing this as something like "Select an *OpenShift* provider." or "Select the checkboxes of the OpenShift providers to configure image scanning on."

File: doc-QuickStart_OpenShift_Provider_Guide/topics/image-scanning.adoc [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+=== Evaluating Compliance
+
+If the image scan policy has been updated since the last scan, compliance conditions may be re-evaluated. From an image summary page, select menu:Policy[Check Compliance of Last Known Configuration]. Refreshing the image page will reflect the latest compliance history.
+
+== OpenSCAP Policy Profile
+
+{product-title} is pre-configured with a default scanning policy profile. This includes conditions to scan and identify compliance, as well as annotate compliance failure. SmartState analysis is performed when new images are added to OpenShift.
+
+Enabling deny-execution annotation (external link)
```
Comment (dayleparker) [inline doc-QuickStart_OpenShift_Provider_Guide/topics/image-scanning.adoc]: This link seems to be missing - would it be possible to check/fix this?

File: doc-QuickStart_OpenShift_Provider_Guide/topics/image-scanning.adoc [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+{product-title} manages vulnerability scanning of container images. When an OpenShift provider is added, OpenShift images from the internal registry are discovered. To enable image scanning, perform the following configuration steps:
+
+. Navigate to menu:Compute[Containers > Providers].
+. Select the *OpenShift* provider checkbox.
+. From the *Policy* pull-down menu, click *Manage Policies*.
+. Select the *OpenSCAP profile* checkbox.
+. Click *Save*.
+
+This action will trigger a SmartState analysis, or scan, of all images referenced by the OpenShift provider. The initial scan may take several hours to complete, depending on the number and size of images. OpenShift scanning pods are scheduled for a maximum of three at a time.
```
Comment (dayleparker) [inline doc-QuickStart_OpenShift_Provider_Guide/topics/image-scanning.adoc]: This sentence is a little confusing: "OpenShift scanning pods are scheduled for a maximum of three at a time." Does it mean: "A maximum of three OpenShift pods can be scanned at a time."? Would you mind rewording just a little please?

**Comment by dayleparker** [comment]
@aweiteka 
Hi Aaron, 
Apologies for the delay in getting to this PR. The section looks excellent -- huge thanks for providing this content to the docs! 
I came across and made a couple small punctuation and style edits for consistency with our other docs (I hope that was OK -- in the interest of saving you time), and I had a couple remaining questions about the wording, and one link that looks like it might be missing. Would you mind doing one last quick check and letting me know how to proceed?
Many thanks,  
Dayle
Would you

File: doc-QuickStart_OpenShift_Provider_Guide/topics/image-scanning.adoc [hunk 4]
```diff
@@ -0,0 +1,80 @@
+== Configuring Image Scanning
+
+{product-title} manages vulnerability scanning of container images. When an OpenShift provider is added, OpenShift images from the internal registry are discovered. To enable image scanning, perform the following configuration steps:
+
+. Navigate to menu:Compute[Containers > Providers].
+. Select the *OpenShift* provider checkbox.
```
Comment (dayleparker) [inline doc-QuickStart_OpenShift_Provider_Guide/topics/image-scanning.adoc]: For this step, I was a little confused when running through the procedure on my test instance. Would it be OK to reword as something like "Select the OpenShift providers to enable scanning on." or "Select the checkboxes of the OpenShift providers to scan." or similar?

File: doc-QuickStart_OpenShift_Provider_Guide/topics/image-scanning.adoc (hunk 1 above)
Comment (aweiteka) [inline doc-QuickStart_OpenShift_Provider_Guide/topics/image-scanning.adoc]: Updated.

File: doc-QuickStart_OpenShift_Provider_Guide/topics/image-scanning.adoc (hunk 2 above)
Comment (aweiteka) [inline doc-QuickStart_OpenShift_Provider_Guide/topics/image-scanning.adoc]: Removed this line. It's covered in the last section.

File: doc-QuickStart_OpenShift_Provider_Guide/topics/image-scanning.adoc (hunk 3 above)
Comment (aweiteka) [inline doc-QuickStart_OpenShift_Provider_Guide/topics/image-scanning.adoc]: Updated

**Comment by aweiteka** [comment]
> Would you mind doing one last quick check and letting me know how to proceed?

Updates pushed. I wasn't sure how this fit into the larger docs set (is this stand-alone?) but assuming all is good there it looks good to me.

**Comment by dayleparker** [comment]
Thanks for the updates, Aaron! I think having the content as part of this guide makes perfect sense. We've also been trying to not add too many more guides to the MIQ/CloudForms suite, as we have quite a few already. :) Thanks again for all your work on this -- much appreciated.

BTW, this merge will put the content into the Fine release/CloudForms 4.5 docs; does it apply to Euwe/CloudForms 4.2 as well? I will backport it if so.

**Comment by aweiteka** [comment]
> does it apply to Euwe/CloudForms 4.2 as well?

I don't think so. @simon3z can you confirm?

**Comment by simon3z** [comment]
> > does it apply to Euwe/CloudForms 4.2 as well?
> 
> I don't think so. @simon3z can you confirm?

@dayleparker @aweiteka scanning container images with OpenSCAP was delivered with 4.2 (~1 year ago). I quickly checked if there was anything 4.5-related but I haven't noticed anything.
So it seems that indeed this is applies to 4.2 as well.

**Comment by dayleparker** [comment]
Awesome, thanks for confirming, @simon3z. I've cherry-picked this over to euwe now in https://github.com/ManageIQ/manageiq_docs/pull/373. Thanks again for your work on this @aweiteka!

**Pull request merged**